management-api-server/src/main/java/com/datastax/mgmtapi/resources/v2/RepairResources.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.resources.v2;
+
+import com.datastax.mgmtapi.ManagementApplication;
+import com.datastax.mgmtapi.resources.common.BaseResources;
+import com.datastax.mgmtapi.resources.v2.models.RepairParallelism;
+import com.datastax.mgmtapi.resources.v2.models.RepairRequest;
+import com.datastax.mgmtapi.resources.v2.models.RepairRequestResponse;
+import com.datastax.mgmtapi.resources.v2.models.RingRange;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/api/v2/repairs")
+public class RepairResources extends BaseResources {
+
+  public RepairResources(ManagementApplication application) {
+    super(application);
+  }
+
+  @PUT
+  @Operation(summary = "Initiate a new repair", operationId = "v2PutRepair")
```

Comment:
Suggestion:
```suggestion
  @Operation(summary = "Initiate a new repair", operationId = "putRepairV2")
```
The `operationId` comes mostly into play when the client is generated. For `getTables()`, we've gone with the template of `<operation>V#`. For all of the v0 and v1 endpoints, the method names are all over the place, so we don't exclusively use the HTTP verbs (GET, PUT, DELETE, etc).

So for v2, I think I'd like to append the version to the end. My reasoning is that when trying to find a method you want to use in a client application (i.e. Reaper), you are more likely going to look for the function you want before looking for the version of it. IDEs tend to give you pop-ups with all the available methods on the generated client, and they are usually in alphabetical order. So having the `v2` prepended to the beginning of the method name will mean that you need to scroll down to find the v2 version specifically. I think it's better to have, for example:
- getKeyspaces()
- getKeyspacesV1()
- getKeyspacesV2()

Having said all that, and going back to my earlier point that we don't consistently use the approrpiate HTTP verbs as the method prefix, this nice neat grouping of similar methods isn't going to happen anyway. Still, I think it would be easier to have the version at teh end of the method name as opposed to prefixing it.

Reply:
OK, I'm happy to go along with this (PS: remember to use issue/suggesiton.